script.js

```diff
@@ -0,0 +1,30 @@
+var hamburgerButton = document.querySelector(".hamburger");
```

Comment:
- [ ] kindly remember JavaScript was used in this project so you need JS linters in the repository for code checks. it will run on your code to find formatting discrepancies, non-adherence to coding standards and conventions, and find logical errors in your program. please [check here](https://github.com/microverseinc/linters-config/tree/master/html-css-js) to see how to go about it.